components/markdown/tests/summary.rs

```diff
@@ -45,3 +42,32 @@ And some content after
     );
     insta::assert_snapshot!(body);
 }
+
+#[test]
+fn truncated_summary() {
+    let body = get_summary(
+        r#"
+Things to do:
+* Program <!-- more --> something
+* Eat
+* Sleep
+    "#,
+    );
+    insta::assert_snapshot!(body);
+}
+
+#[test]
+fn foontnotes_summary() {
```

Comment:
nit: maybe you meant this (similar change for snap file)
```suggestion
fn footnotes_summary() {
```